README.md

```diff
@@ -1 +1,40 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+This is a mobile version skeleton of Portfolio.
+![Screenshot from 2022-02-16 17-06-35](https://user-images.githubusercontent.com/73285210/154257149-471f4cb2-fa17-4146-8ec8-397cc7cb1dd3.png)
+
+
```

Comment:
- [ ] Great job, you have added an image to your readme, awesome it is so professional, but I think the image you have chosen is not the appropriate one, I will drop an image down to show you how you can take a screenshot of your application a clean way, take another picture and change update your readme, please.
<img width="428" alt="tof" src="https://user-images.githubusercontent.com/79058364/154266149-2dd2d339-55d6-48df-97c7-a692083dfcde.png">